templates/index.ts.hbs

```diff
@@ -1,5 +1,5 @@
 {{#each children}}
-export { {{name.upperCamelCase}} } from "./{{name.lowerCamelCase}}";
+export { {{name.upperCamelCase}}, paramKeys as {{name.upperCamelCase}}ParamKeys } from "./{{name.lowerCamelCase}}";
```

Comment:
I wonder if a better interface would be to provide `ShopperProductsParamKeys` and `ShopperProductsParamKeysRequired` so it's easier to consume the types and avoid using template strings in the commerce-sdk-react like:

```
paramKeysMap[`${methodName}Required`]
```

Reply:
The template string above is actually not necessary. I was following the pattern that already exists in the file. But we can access the parameter keys directly if we want to.